### DISCUSSION TITLE
📋The Algorithm of BeWater Buidler Board📋!

### DISCUSSION THREAD

**Discussion by leeduckgo** [BeWaterXYZ/bewater-frontend#552] [Features]
We are designing buidler board for [bewater.xyz](https://bewater.xyz) !

> 🤔 What do you think about the algorithm design of buidler board ?

Should we use the number of stars⭐️ as a criterion for sorting developers and repositories? Or......

You have a better idea💡🤩? 

🏈🏈🏈 Show me ur cooooool idea in this post, get involved in the BeWater project! 🏈🏈🏈

**Comment by lukema95** [comment]
For evaluating projects, I suggest considering the following dimensions and their corresponding weights:

- ✨Star count (3/10): Indicates the project's popularity and recognition
- Fork count (2/10): Reflects how many developers have referenced or built upon the project
- Commit count (2/10): Measures the project's development activity level
- Contributor count (2/10): Shows the strength and diversity of the project's community
- Last commit date (1/10): Indicates whether the project is actively maintained

These metrics collectively provide insights into a project's popularity, adoption rate, development activity, community engagement, and maintenance status.

**Reply by leeduckgo** [reply]
cool! ( ͡° ͜ʖ ͡°)

**Comment by benschiller** [comment]
Great suggestions @lukema95 🙌

We could also consider adding a focus on the quality of contributions. For example, rewarding helpful code reviews, merged pull requests, and extra weight for solving high-priority issues would highlight impact of action and meaningful contributions 👍

**Reply by leeduckgo** [reply]
Agree! We are considering use BeWater as the example, the contributors for BeWater projects will be marked in the BeWater's user system!
